code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -131,11 +131,11 @@ var/list/wizard_snakes
 		wizard_snakes = list()
 	wizard_snakes.Add(src)
 
-/mob/living/simple_animal/cat/snek/wizard/Die()
+/mob/living/simple_animal/cat/snek/wizard/death_untransmogrify()
```

Comment:
God damnit I fixed this on another computer and completely forgot about it due to compile times.

Dear god I'm just now remembering how painful DM is.